api-guidelines/v4.0/API-DESIGN-GUIDELINES/RESOURCES.md

```diff
@@ -57,43 +67,83 @@ School Year, and a reference to a School. Resources _must_ be accessible by
 primary key values using the standard HTTP GET query string search syntax:
 
 ```none
-{resourceURI}?keyField1={value1}&keyField2={value2}
+{resourceURI}?{keyField1}={value1}&{keyField2}={value2}
```

Comment:
Ugh, the school year is a horrible legacy IMHO. It was explained to me years ago, but I still fail to understand, why it is "schoolYearType" instead of "schoolYear". And then yes there are aliases. You bring up a great point that is worth exploring. I'll give some thought to how this could be done.